README.md

```
@@ -11,10 +11,28 @@ To make a change to the node-red runtime being used by K4 avalanche:
+- `arch -x86_64 npm pack` - this generates a `tgz` file
+
+In your local copy of k4avalanche reference this file in your `package.json`:
+
+- `"@servisbot/node-red": "file:../node-red/servisbot-node-red-0.18.7-patch-15.tgz"`
+- `npm install`
+
+Now when you start k4, it will include your local copy of node-red.
 
 # CHANGE-LOG
 
+## 0.18.8
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)